### PR TITLE
feat: approval - requested reviewers

### PR DESCRIPTION
**Pull request by shine2lay**
Add a sub option `requested_reviewers` for approvals
for more context, see #215 

Note: this only implement the latter part of #215 

image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ For convenience, wildcards can be used: `pull_request.*`, `issues.*`, `pull_requ
     reviewers: [ user1, user2 ] # list of github usernames required to review
     owners: true # Optional boolean. When true, the file .github/CODEOWNER is read and owners made required reviewers
     assignees: true # Optional boolean. When true, PR assignees are made required reviewers.
+    pending_reviewer: true #Optional boolean. When true, all the reviewer's approval is
```
Comment (jusx) [inline docs/README.md]: Missing a space after the `#` -- Also looks like an intentional unfinished sentence?

File: lib/validators/approvals.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ class Approvals extends Validator {
       'pull_request.unlabeled',
       'pull_request.synchronize'
     ]
+    this.supportedEvents.push('block')
```
Comment (jusx) [inline lib/validators/approvals.js]: Should this be `pull_request.block` ? Thought I couldn't find such an event in the API docs...

Comment (shine2lay) [reply lib/validators/approvals.js]: this should not be there, it was my mistake

File: docs/README.md (hunk 1 above)
Comment (shine2lay) [reply docs/README.md]: fixed

**Pull request merged**